src/store/modules/wallets.js

```diff
@@ -237,104 +70,26 @@ export function simplifyWallet(walletId: WalletId, isSimplified: boolean) {
   }
 }
 
-export function clean() {
-  return {
-    type: CLEAN,
-  }
-}
-
 export type WalletsAction =
-  ExtractReturn<typeof goToStartView> |
-  ExtractReturn<typeof openView> |
-  ExtractReturn<typeof closeView> |
-  ExtractReturn<typeof changeNameInput> |
-  ExtractReturn<typeof changePasswordInput> |
-  ExtractReturn<typeof changePasswordHintInput> |
-  ExtractReturn<typeof changePasswordConfirmInput> |
-  ExtractReturn<typeof setWallets> |
   ExtractReturn<typeof setWalletsItems> |
   ExtractReturn<typeof setActiveWallet> |
-  ExtractReturn<typeof setIsLoading> |
-  ExtractReturn<typeof setInvalidField> |
   ExtractReturn<typeof privateKeyError> |
   ExtractReturn<typeof privateKeySuccess> |
   ExtractReturn<typeof privateKeyRequest> |
-  ExtractReturn<typeof simplifyWallet> |
-  ExtractReturn<typeof clean>
+  ExtractReturn<typeof simplifyWallet>
```

Comment:
implementation of `simplifyWallet` already exist?

Reply:
yes